### ISSUE TITLE
Add -OutputPath to Build-Bicep

### ISSUE THREAD

**Issue by StefanIvemo** [StefanIvemo/BicepPowerShell#109]
**Is your feature request related to a problem? Please describe.**
We should have an equivalent to `--outfile` that exists in Bicep CLI
`--outfile <file>  Saves the output as the specified file path.`

**Describe the solution you'd like**
```powershell
Build-Bicep -Path .\vnet.bicep -OutputPath .\virtualNetork.json
```

**Additional context**

**Comment by PalmEmanuel** [comment]
A suggestion is to instead change it to -OutputPath and combine it with the current parameter for a specified directory.

**Comment by emilguden** [comment]
Possible solution provided in PR #130

**Comment by StefanIvemo** [comment]
Closed by #130
